.github/workflows/ccpp.yml

```diff
@@ -11,6 +11,16 @@ jobs:
     - name: build examples and tests
       run: rm -rf build || true && mkdir build && cd build && cmake .. -DLIBMBUS_BUILD_EXAMPLES=ON -DLIBMBUS_BUILD_TESTS=ON && cmake --build . -j
 
+  test:
```

Comment:
Since this change is reverted in the next commit without any explanation, please drop them.